lib/Helper/UserHelper.php

```diff
@@ -69,10 +69,9 @@ public function getGroupIdsForUser(string $userId): ?array {
 			return null;
 		}
 
-		$groupArray = [];
-		foreach ($userGroups as $group) {
-			$groupArray[] = $group->getGID();
-		}
+		$groupArray = array_map(function (IGroup $group) {
+			return $group->getGID();
+		}, $userGroups);
```

Comment:
Looks nicer, but is heavier on RAM. Not a blocker.

Reply:
Then this message is redirect to me. With small data sized it's not that significant. With groups, could be so so.